aspnetcore/tutorials/publish-to-azure-webapp-using-vs.md

```diff
@@ -180,6 +180,7 @@ When you have finished testing the app, go to the [Azure portal](https://portal.
 
 ## Additional resources
 
+* For Visual Studio Code, see [Folder publish](https://github.com/aspnet/websdk#folder-publish)
```

Comment:
We have that in the doc if you want ... https://docs.microsoft.com/aspnet/core/host-and-deploy/visual-studio-publish-profiles#publish-profiles

```suggestion
* For Visual Studio Code, see [Publish profiles](xref:host-and-deploy/visual-studio-publish-profiles#publish-profiles)
```